### PR TITLE
Improvements: FE boosts up loading process state

### PR DESCRIPTION
**Pull request by lciolecki**

### CODE REVIEW

File: ui/client/containers/admin/CustomProcesses.js [hunk 1]
```diff
@@ -40,13 +40,13 @@ class CustomProcesses extends BaseProcesses {
 
   deploy = (process) => () => {
     this.props.actions.toggleConfirmDialog(true, DialogMessages.deploy(process.name), () => {
-      return HttpService.deploy(process.name).then(() => this.reload())
+      return HttpService.deploy(process.name).finally(() => this.reload(true))
```
Comment (arkadius) [inline ui/client/containers/admin/CustomProcesses.js]: What is this argument of reload? I see in code only non-argument version of this method. Why you don't do something like `this.props.actions.loadProcessState`?

File: ui/client/reducers/graph.js [hunk 2]
```diff
@@ -16,6 +16,8 @@ const emptyGraphState = {
   processCounts: {},
   testResults: null,
   businessView: false,
+  processState: null,
+  stateLoaded: false,
```
Comment (arkadius) [inline ui/client/reducers/graph.js]: please rename to processStateLoaded

Comment (lciolecki) [reply ui/client/reducers/graph.js]: Changed

File: ui/client/containers/admin/CustomProcesses.js (hunk 1 above)
Comment (lciolecki) [reply ui/client/containers/admin/CustomProcesses.js]: Changed

**Pull request merged**